jep/7/README.adoc

```diff
@@ -132,11 +183,106 @@ Workarounds for this include downloading affected plugins from Artifactory, and
 
 If previous core compatibility fixes are reverted, or future core changes break ruby-runtime, users of those plugins will be impacted.
 
+=== Affected plugins, Ruby runtime
+
+Below you can find a list of the affected plugins which are/were being hosted in the main Jenkins update center.
+There might be other 3rd-party plugins affected.
+
+Gitlab Hook::
+Last released **5 years ago**. +
+Contains multiple security vulnerabilties. +
+Suggestion: Use the plugin:gitlab-plugin[GitLab] plugin and the plugin:gitlab-branch-source[GitLab Branch Source] plugin.
+
+Cucumber::
+Last released **8 years ago**. +
+Suggestion: Use `sh` or `bat` to run `cucumber` from the command line.
+
+pyenv::
+Last released **7 years ago**. +
+Suggestion: Use `sh` or `bat` to run `pyenv` from the command line.
+
+Rvm::
+Last released **5 years ago**. +
+Suggestion: Use `sh` or `bat` to run `rvm` from the command line.
+
+Capitomcat::
+Last released **6 years ago**. +
+Suggestion: Install Ruby and Capistrano and use `sh` or `bat` to invoke them from the command line.
+
+Commit Message Trigger::
+Last released **7 years ago**. +
+Suggestion: Use `sh`, `bat`, or other scripts to read git commit messages and conditionally execute Pipeline steps.
+
+Git notes::
+Last released **9 years ago**. +
+Suggestion: Use `sh`, `bat`, or other scripts to run `git` to annotate commits.
+
+rbenv::
+Last released **5 years ago**. +
+Suggestion: Use `sh` or `bat` to run `rbenv` from the command line.
+
+Chef::
+Last released **6 years ago**. +
+Suggestion: Use `sh` or `bat` to run `chef` from the command line.
+
+CI Skip::
+Last released **7 years ago**. +
+Suggestion: Use the plugin:github-scm-trait-commit-skip[GitHub Commit Skip SCM Behaviour], plugin:bitbucket-scm-trait-commit-skip[Bitbucket Commit Skip SCM Behaviour], or plugin:scmskip[SCM Skip] to skip builds based on the content of commit messages.
```

Comment:
Bad copy and paste doesn't work outside jenkins.io

Reply:
Ack, will fix in a follow-up

Reply:
```suggestion
Suggestion: Use the https://github.com/jenkinsci/github-scm-trait-commit-skip-plugin[GitHub Commit Skip SCM Behaviour], https://github.com/jenkinsci/bitbucket-scm-trait-commit-skip-plugin[Bitbucket Commit Skip SCM Behaviour], or https://github.com/jenkinsci/scmskip-plugin[SCM Skip] to skip builds based on the content of commit messages.
```